In [1]:
import pandas as pd
import numpy as np
from scipy import stats
import scipy
from scipy.stats import chi2_contingency

In [2]:
df = pd.read_csv('BuyerRatio.csv',index_col=0)
df

East  West  North  South
Observed Values                          
Males              50   142    131     70
Females           435  1523   1356    750

null hypothesis = All proportions are equal

Alternate hypothesis = Not all proportions are equal

In [3]:
chi,p,dof,exp = chi2_contingency(df)
p


0.6603094907091882

In [4]:
if p>0.05:
    print(f"Since P:{p} > alpha: 0.05 Failed to reject null hypothesis")
else:
    print(f"Since P:{p} < alpha: 0.05 Reject null Hypothesis")

Since P:0.6603094907091882 > alpha: 0.05 Failed to reject null hypothesis


Since P:0.6603094907091882 > alpha: 0.05 Failed to reject null hypothesis

All Proportions are equal